Extensions/xkit_preferences.js

```diff
@@ -1590,263 +1590,266 @@ XKit.extensions.xkit_preferences = new Object({
 
 	},
 
-	return_extension_settings: function(extension_id) {
```

Comment:
I'm on mobile so I can't check this out right now, but what happened to the diff here?